proto/cosmos/group/v1beta1/types.proto

```diff
@@ -221,16 +218,16 @@ message Proposal {
     option (gogoproto.goproto_enum_prefix) = false;
 
     // An empty value is not allowed.
-    EXECUTOR_RESULT_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "ProposalExecutorResultInvalid"];
+    EXECUTOR_RESULT_UNSPECIFIED = 0;
```

Comment:
should also prefix this with `PROPOSAL_`? it's a bit longer but more consistent with the rest